common/beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -143,12 +143,6 @@ bool mon_wlan_hal_dummy::sta_beacon_11k_request(const SBeaconRequest11k &req, in
     return true;
 }
 
-bool mon_wlan_hal_dummy::sta_statistics_11k_request(const SStatisticsRequest11k &req)
```

Comment:
According to the [commits ](https://github.com/prplfoundation/prplMesh/blob/master/CONTRIBUTING.md#commits)section in CONTRIBUTING.md: "*Every commit must stand on its own. No commit should ever break something.*"

If you remove a method that is pure virtual in the base class and overridden in derived classes, you must remove them all at once. Put in other words, remove sta_statistics_11k_request from all dummy, dwpal and nl80211 libraries in the same commit. Otherwise the commit is not complete (code must compile without errors after every single commit).

Reply:
fixed